keras_rs/src/utils/loss_and_metric_utils.py

```diff
@@ -0,0 +1,62 @@
+from typing import Optional
```

Comment:
I'm not a huge fan of the concept of a `utils` folder in general, and I typically prefer the utils files to be in the same folder as where it's used. This one is a bit funny because it's for both metrics and losses, but I think it wouldn't be too shocking to put it in `keras_rs/src/metrics` since a loss is just a kind of metric.

What do you think? Note that keras itself is not consistent on this.

Reply:
Yeah, I agree. I'm not a huge fan of a `utils` folder either, unless it has very generic ops (like checking shape, etc.), which might be used across the repo. I'll move this